packages/api-client/src/components/ApiClient/Request/Request.vue

```diff
@@ -7,7 +7,13 @@ import RequestHeaders from './RequestHeaders.vue'
 import RequestQuery from './RequestQuery.vue'
 import RequestVariables from './RequestVariables.vue'
 
-const { activeRequest, readOnly } = useRequestStore()
+const { activeRequest } = useRequestStore()
+
+/**
+ * TODO: This is a workaround to make the address bar editable, but not the rest. If we’d really like to have an
+ * API client where everything can be edited, we need to invest more time.
+ */
+const readOnly = true
```

Comment:
Currently, we don’t want the operation name to be editable in any case (because it’s not really tied up anyway).